ruma-api-macros/src/util.rs

```diff
@@ -257,3 +257,11 @@ pub(crate) fn req_res_name_value<T>(
     *header = Some(value);
     Ok(field_kind)
 }
+
+fn is_ascii_printable(b: &u8) -> bool {
+    b.wrapping_sub(b' ') < 0x5F
+}
+
+pub(crate) fn is_printable(string: &str) -> bool {
```

Comment:
`is_printable` is very misleading if it actually checks whether all characters are printable *ASCII* chars. I'd rename this `is_ascii_printable` and inline the helper function above.